cmake/OpenSimMacros.cmake

```diff
@@ -253,3 +253,29 @@ function(OpenSimAddApplication APPNAME)
     endif()
 
 endfunction()
+
+
+# Function to copy DLL files from dependency install directory into OpenSim 
+# build and install directories. This is a Windows specific function enabled 
+# only for Windows platform. Intention is to allow runtime loader to find all 
+# the required DLLs without need for editing PATH variable.
```

Comment:
Awesome function. One issue we've had in the past with copying dll's is we used to GLOB all the libraries in a directory and this ended up copying ALL libraries on UNIX machines (if Simbody was installed into `/usr/local` or something). Since this function only does anything on windows, this is unlikely to be an issue. A slightly more robust solution would be to loop through all the libraries that a package exports. I think this would be possible, but is unnecessary for now. I'm just mentioning this to consider if we run into problems in the future.